apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php

```diff
@@ -316,25 +316,25 @@ function enqueue_coblocks_gallery_scripts() {
 add_action( 'wp_enqueue_scripts', __NAMESPACE__ . '\enqueue_coblocks_gallery_scripts' );
 
 /**
- * Load Homepage Posts block.
+ * Load Blog Posts block.
  */
-function load_homepage_posts_block() {
+function load_blog_posts_block() {
 	$disable_block = (
-		in_array( 'newspack-blocks/newspack-blocks.php', (array) get_option( 'active_plugins', [] ), true ) ||
-		in_array( 'newspack-blocks/newspack-blocks.php', (array) get_site_option( 'active_sitewide_plugins', [] ), true )
+		in_array( 'newspack-blocks/newspack-blocks.php', (array) get_option( 'active_plugins', array() ), true ) ||
+		in_array( 'newspack-blocks/newspack-blocks.php', (array) get_site_option( 'active_sitewide_plugins', array() ), true )
 	);
 
 	/**
-	 * Can be used to disable the Homepage Posts block.
+	 * Can be used to disable the Blog Posts block.
 	 *
 	 * @since 0.15.1
 	 *
-	 * @param bool $disable_block True if Homepage Posts block should be enabled, false otherwise.
+	 * @param bool $disable_block True if Blog Posts block should be enabled, false otherwise.
 	 */
-	if ( apply_filters( 'a8c_disable_homepage_posts_block', $disable_block ) ) {
+	if ( apply_filters( 'a8c_disable_blog_posts_block', $disable_block ) ) {
 		return;
 	}
 
-	require_once __DIR__ . '/homepage-posts/index.php';
+	require_once __DIR__ . '/blog-posts/index.php';
```

Comment:
⚠️ This path is wrong: `failed to open stream` :

```
[04-Dec-2019 18:45:40 UTC] PHP Warning:  require_once(/var/www/html/wp-content/plugins/full-site-editing-plugin/blog-posts/index.php): failed to open stream: No such file or directory in /var/www/html/wp-content/plugins/full-site-editing-plugin/full-site-editing-plugin.php on line 338
```

Reply:
Resolved in 5138e90